flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/kitversion/KitVersionJpaRepository.java

```diff
@@ -17,8 +17,11 @@ public interface KitVersionJpaRepository extends JpaRepository<KitVersionJpaEnti
     @Modifying
     @Query("""
             UPDATE KitVersionJpaEntity k
-            SET k.status = :status
+            SET k.status = :status,
+                k.statusVersion = :statusVersion
             WHERE k.id = :id
         """)
-    void updateStatus(@Param("id") long kitVersionId, @Param("status") int status);
+    void updateStatus(@Param("id") long kitVersionId,
+                      @Param("status") int status,
+                      @Param("statusVersion") Long statusVersion);
```

Comment:
**Suggestion:** Ensure that the `statusVersion` parameter in the `updateStatus` method is not null to prevent potential null pointer exceptions when updating the database. [possible issue, importance: 8]
```suggestion
                      void updateStatus(@Param("id") long kitVersionId, @Param("status") int status, @Param("statusVersion") long statusVersion);
```

Reply:
سه تا کامنت روی این خط داده که من تفاوت‌شون رو تشخیص نمی‌دم.